### PR TITLE
Tweaks Service Pistol stats

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request

* Mag size 10 -> 12
* Magazine sprite tweak to match new capacity
* Burn per shot 10 -> 11
* Extra magazine in every head locker (HoS had to lose box of flashbangs for this)

## Why It's Good For The Game

Service pistol aimed to be a minor improvement over the PTSD but failed to accomplish this by being substantially weaker both as a stun weapon and as a lethal weapon. The tweaked stats mentioned above allow the service pistol to:

* Have the same number of shots as PTSD did in disabler mode, while having a hybrid damage split that makes them weaker as a disabler
* Have an extra clip available to make the overall output actually higher than PTSD, instead of roughly the same. Still substantially lower DPS than other energy weapons
* Have just a little more oomph when it comes to the lethal output vs simplemobs and such.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

image

image

</details>

## Changelog
:cl:
balance: Service pistol has modestly improved stats: 12 shots up from 10, 10% more burn damage, and an extra magazine is now available in all head lockers. 
tweak: Adjusted service pistol magazine sprite to accommodate increased capacity.
tweak: Box of flashbangs has been removed from HoS locker to make room for the extra magazine. 
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
I originally added a light overlay but I cba to fix the fact they don't work with non-energy pistols.

**Comment by Rukofamicom** [comment]
I like how the new hybrid damage type plays out, but I do think this would still be cooler as a self-charging mag

**Comment by MarkusLarsson421** [comment]
> I do think this would still be cooler if the magazines were slowly self-charging 

The slow self-recharge feature was so slow it was painful. Missed all your shots? Now you got to wait for 10 whole minutes for it to recharge! That's what it feels like at least. It was really painful.

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -40,8 +40,8 @@
 		impact_effect_type = /obj/effect/temp_visual/impact_effect/red_laser/wall
 
 /obj/projectile/beam/laser/lesslethal
-	damage = 10
-	stamina = 18
+	damage = 11
+	stamina = 25
```
Comment (PowerfulBacon) [inline code/modules/projectiles/projectile/beams.dm]: Stamian damage should be strictly lower than the stamina damage output of the disabler. All other things like increased ammo are fine, I was planning to put rechargers back in the heads offices.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/beams.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/projectiles/projectile/beams.dm]: I'd rather the DPS be lower and they be given backup clips or the capability to recharge instead.

Comment (Rukofamicom) [inline code/modules/projectiles/projectile/beams.dm]:
```suggestion
	stamina = 20
```

Comment (Rukofamicom) [reply code/modules/projectiles/projectile/beams.dm]: One extra magazine has been added to each locker. I still don't like the idea of being able to create or order additional mags personally, but if it's limited to a single one for heads it should be fine.

**Comment by Rukofamicom** [comment]
Well fuck
Now the HoS locker is overloaded.

**Comment by Therealdoooc213** [comment]
Maybe you could remove the flashbang box instead of the telescopic shield? I don't think I've seen anyone take the flashbang box and the riot shield is very useful.

**Comment by Rukofamicom** [comment]
That was my other choice and Bacon agrees. 

Both flashbangs and riot shields are available in the rest of sec, so the swap is inconsequential to me.

**Review by PowerfulBacon — APPROVED** [review]
Seems good

**Pull request merged**